IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -85,6 +86,91 @@ - (void)setCallback:(MSIDLogCallback)callback
     });
 }
 
+- (MSIDLogLevel)level
+{
+    if (self.loggerConnector) return self.loggerConnector.level;
+    
+    return _level;
+}
+
+- (BOOL)PiiLoggingEnabled
+{
+    BOOL result;
+    @synchronized (self)
```

Comment:
will this introduce any additional performance hit compared to current logic? (I'm not sure if this property is even used anywhere in our code).

Reply:
No performance hit, atomic properties already do the same internally. If we don't need this property to be atomic, we can remove the lock and simplify code.

Reply:
ok, I didn't notice it was atomic before. Seems unnecessary to me :)